demisto_sdk/commands/validate/tools.py

```diff
@@ -32,3 +34,66 @@ def collect_all_inputs_from_inputs_section(content_item: Playbook) -> Set[str]:
     inputs: dict = content_item.data.get("inputs", {})
     inputs_keys = [input["key"].strip() for input in inputs if input["key"]]
     return set(inputs_keys)
+
+
+def filter_by_marketplace(marketplaces, pack_meta_file_content, return_is_valid=True):
+    """Filtering pack_metadata tags by marketplace"""
+
+    pack_tags: Dict[str, List[str]] = {}
+    for marketplace in marketplaces:
+        pack_tags[marketplace] = []
+    pack_tags["common"] = []
+
+    is_valid = True
+    for tag in pack_meta_file_content.get("tags", []):
+        if ":" in tag:
```

Comment:
can it be done with regex?

Reply:
I think it's a simple check, not worth adding a regex to it.